### DISCUSSION TITLE
Using remote  rabbitmq server rather than local host

### DISCUSSION THREAD

**Discussion by mkassir95** [rabbitmq/rabbitmq-server#6528] [Other] [answered]
I have a spring project, and I want to use remote rabbitmq server(example:https://trail.test.com) rather than local host

> spring.rabbitmq.host=trail.test.com
spring.rabbitmq.port=5672
spring.rabbitmq.username=guest
spring.rabbitmq.password=guest

But I am getting this error in postman `java.net.ConnectException: Connection timed out: no further information`

**Comment by michaelklishin** [comment, marked as answer]
Please never use GitHub issues for questions. Discussions have been a public feature for over a year now.

By default you cannot connect with those default well-known credentials from remote hosts for [obvious security reasons](https://www.rabbitmq.com/access-control.html#loopback-users). You have to create a new user, grant it access to the virtual host in question and use those credentials.

RabbitMQ logs [all connection lifecycle events](https://www.rabbitmq.com/logging.html#logged-events) for connections that send at least one byte of data.
